README.rst

```diff
@@ -503,6 +503,16 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+SNMP DeviceMonitoring
+~~~~~~~~~~~~~~~~~~~~~
+
+This check provides an alternative protocol to collect monitoring information from devices that don't have
+the ability to use `openwisp-config agent <https://github.com/openwisp/openwisp-config/>`_.
```

Comment:
the agent is https://github.com/openwisp/openwrt-openwisp-monitoring, we should refer to it as the OpenWISP Monitoring Agent.